applications/StructuralMechanicsApplication/tests/cpp_tests/test_mpc_plate_tension.cpp

```diff
@@ -0,0 +1,164 @@
+    //Add DOF
+    VariableUtils().AddDofWithReaction(DISPLACEMENT_X, REACTION_X, model_part);
+    VariableUtils().AddDofWithReaction(DISPLACEMENT_Y, REACTION_Y, model_part);
+    // Initialize elements
+    auto& r_process_info = model_part.GetProcessInfo();
+    for (auto& elem : model_part.Elements())
+    {
+        elem.Initialize();
+        elem.InitializeSolutionStep(r_process_info);
+    }
+    // Set initial solution
+    for (auto& node : model_part.Nodes())
+    {
+        (node.FastGetSolutionStepValue(DISPLACEMENT)).clear();
+    }
+    //Apply BC
+    pnode1->Fix(DISPLACEMENT_X);
+    pnode1->Fix(DISPLACEMENT_Y);
+    pnode2->Fix(DISPLACEMENT_Y);
+    pnode4->Fix(DISPLACEMENT_X);
+    //Creat Load
+    auto p_condition1 = model_part.CreateNewCondition("LineLoadCondition2D2N", 1, std::vector<IndexType>({2,6}), p_Properties);
+    auto p_condition2 = model_part.CreateNewCondition("LineLoadCondition2D2N", 2, std::vector<IndexType>({6,3}), p_Properties);
+    array_1d<double, 3> load;
+    load[0] = 1000.0; load[1] = 0.0; load[2] = 0.0;
+    p_condition1->SetValue(LINE_LOAD, load);
+    p_condition2->SetValue(LINE_LOAD, load);
+    //Apply Constraints
+    std::vector< Dof<double>::Pointer >slave_dofs(2);
+    slave_dofs[0] = pnode8->pGetDof(DISPLACEMENT_X);
+    slave_dofs[1] = pnode8->pGetDof(DISPLACEMENT_Y);
+    std::vector< Dof<double>::Pointer >master_dofs(4);
+    master_dofs[0] = pnode7->pGetDof(DISPLACEMENT_X);
+    master_dofs[1] = pnode5->pGetDof(DISPLACEMENT_X);
+    master_dofs[2] = pnode7->pGetDof(DISPLACEMENT_Y);
+    master_dofs[3] = pnode5->pGetDof(DISPLACEMENT_Y);
+    Matrix relation_matrix = ZeroMatrix(2,4);
+    relation_matrix(0,0) = 0.5;
+    relation_matrix(0,1) = 0.5;
+    relation_matrix(1,2) = 0.5;
+    relation_matrix(1,3) = 0.5;
+    Vector constant_vector = ZeroVector(1);
+    model_part.CreateNewMasterSlaveConstraint("LinearMasterSlaveConstraint", 1, master_dofs, slave_dofs, relation_matrix, constant_vector);
+    //Setting up System
+    SchemeType::Pointer p_scheme = SchemeType::Pointer( new ResidualBasedIncrementalUpdateStaticSchemeType() );
+    LinearSolverType::Pointer p_solver = LinearSolverType::Pointer( new SkylineLUFactorizationSolverType() );
+    BuilderAndSolverType::Pointer p_builder_and_solver = BuilderAndSolverType::Pointer( new ResidualBasedBlockBuilderAndSolverType(p_solver) );
+    SolvingStrategyType::Pointer p_strategy = SolvingStrategyType::Pointer( new ResidualBasedLinearStrategyType(model_part, p_scheme, p_solver, p_builder_and_solver, true));
+    p_strategy->SetEchoLevel(0);
+    p_strategy->Solve();
+    //Output to GID
+    GidIO<> gid_io("TEST_MPC_Plate_Tension", GiD_PostBinary, SingleFile, WriteUndeformed,  WriteElementsOnly);
+    const int nl_iter = model_part.GetProcessInfo()[NL_ITERATION_NUMBER];
+    const double label = static_cast<double>(nl_iter);
+    gid_io.InitializeMesh(label);
```

Comment:
Please comment GidIO and the std::cout

Reply:
did in https://github.com/yamanatoo/Kratos/commit/1a9d34e58552f8b2bb969235d8d4293833765575